plugin/rewrite/reverter_test.go

```diff
@@ -0,0 +1,54 @@
+package rewrite
+
+import (
+	"testing"
+
+	"github.com/coredns/coredns/plugin"
+	"github.com/coredns/coredns/plugin/pkg/dnstest"
+	"github.com/coredns/coredns/plugin/test"
+
+	"github.com/miekg/dns"
+	"golang.org/x/net/context"
+)
+
+func TestResponseReverter(t *testing.T) {
+	rules := []Rule{}
+	r, _ := newNameRule("stop", "regex", "(core)\\.(dns)\\.(rocks)", "{2}.{1}.{3}", "answer", "name", "(dns)\\.(core)\\.(rocks)", "{2}.{1}.{3}")
```

Comment:
use backticks for the regexps: "(core)\\.(dns)\\.(rocks)" -> `(core)\.(dns)\.(rocks)` (markdown doesn't show - but the later uses `)